modules/core/src/main/java/org/apache/ignite/internal/visor/consistency/VisorConsistencyRepairTask.java

```diff
@@ -117,73 +125,84 @@ protected VisorConsistencyRepairJob(VisorConsistencyRepairTaskArg arg, boolean d
             log.info("Consistency check started " +
                 "[grp=" + grpCtx.cacheOrGroupName() + ", part=" + p + ", strategy=" + strategy + "]");
 
-            StringBuilder sb = new StringBuilder();
-
-            sb.append("[node=").append(ignite.localNode());
-            sb.append(", cacheGroup=").append(grpCtx.cacheOrGroupName());
-            sb.append(", part=").append(p).append("]");
-
-            String statusKey = sb.toString();
+            String statusKey = "[node=" + ignite.localNode() + ", cacheGroup=" + grpCtx.cacheOrGroupName() + ", part=" + p + "]";
 
             if (VisorConsistencyStatusTask.MAP.putIfAbsent(statusKey, "0/" + part.fullSize()) != null)
```

Comment:
This isn't an one-liner conditional branch, so it should be in curly braces

Reply:
Fixed